src/software/simulated_tests/validation/terminating_function_validator_test.cpp

```diff
@@ -104,7 +108,7 @@ TEST(TerminatingFunctionValidatorTest,
                                                 ValidationCoroutine::push_type& yield) {
         while (world->ball().position().x() < 0)
         {
-            yield();
+            yield("Ball x position not less than 0");
```

Comment:
nit: Shouldn't this be `"Ball x position less than 0"` instead? Same with the other yields?

Reply:
no, we're saying here that the terminating validation function isn't complete because of the error message given, i.e. The ball's x position is not less than 0